geospatial/dted.ksy

```diff
@@ -0,0 +1,603 @@
+        size: 2
+        doc: |
+          Number of coordinates in accuracy sub region outline. (Maximum of |
+          14 coordinate pairs. The first coordinate is the most southwestern. |
+          Coordinates are input clockwise. Implied closing from last to first |
+          coordinate pairs.) (03-14)
+      - id: coordinate_pair
+        type: coordinate_pair_ddmmss_sh
+        repeat: expr
+        repeat-expr: 14
+        doc: |
+          Start of Coordinate Pair Description. Repeat to maximum of fourteen |
+          times to outline subregion. Blank fill all unused accuracy |
+          subregions and unused portions of subregions.
+  data_record:
+    doc: |
+      Data Record Description. The Data Record is used for magnetic tape and |
+      CD-ROM. |
+      Each elevation is a true value as determined by the Earth Gravity |
+      Model (EGM) 1996 recorded to the nearest meter. The horizontal position |
+      is referenced to precise longitude-latitude locations in terms of the |
+      current World Geodetic System (WGS), determined for each file by |
+      reference to the origin at the southwest corner. The elevations are |
+      evenly spaced in latitude and longitude at the interval designated in |
+      the user header label in South to North profile sequence.
+    seq:
+      - id: recognition_sentinal
+        type: u1
+        doc: |
+          Recognition Sentinel.
+      - id: data_block_count
+        type: b24
+        doc: |
+          Sequential count of the block within the file, starting with zero |
+          for the first block (Fixed Binary). (Data block count)
+      - id: logitude_count
+        type: u2
+        doc: |
+          Count of the meridian. True longitude = longitude count x data |
+          interval + origin (Offset from the SW corner longitude) |
+          (Fixed Binary).
+      - id: latitude_count
+        type: u2
+        doc: |
+          Count of the parallel. True latitude = latitude count x data |
+          interval + origin (Offset from the SW corner latitude) |
+          (Fixed Binary).
+      - id: elevations
+        type: data_record_elevation
+        repeat: expr
+        repeat-expr: _parent.uhl.number_of_latitude_points.value
+        doc: |
+          True elevation value of point N of meridian in meters (Fixed Binary).
+      - id: checksum
+        type: s4
+        doc: |
+          Algebraic addition of contents of block. Sum is computed as an 
+          integer summation of 8-bit values (Fixed Binary).
+  data_record_elevation:
+    doc: |
+      type which repesents a single elevation value with it's unique encoding
+      mechanism.
+    seq:
+      - id: is_negative
+        type: b1
+      - id: value
+        type: b15
+    instances:
+      meters:
+        value: value * (is_negative? -1 : 1)
+  ascii_int:
+    seq:
+      - id: text
+        type: str
+        size: 4
+    instances:
+      value:
+        value: text.to_i
+  coordinate_pair_ddmmss_sh:
```

Comment:
Is `Not Available (NA)` properly handled?

Reply:
No it is not.  I'm not sure how to handle it.  How about something like this:
```
  ascii_int_maybe:
    seq:
      - id: text
        type: str
        size: 4
    instances:
      has_value:
        value: text != "NA"
      value:
        value: text.to_i
```

Reply:
```yaml
value:
  value: text.to_i
  if: has_value
```

Reply:
Done. I'll update to add the `if: has_value`